engines/osxnative/dscl.go

```diff
@@ -0,0 +1,65 @@
+package osxnative
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+)
+
+type dscl struct {
+}
+
+// Run dscl
```

Comment:
I think it's usually common to document a method like this:
// run will exectue a dscl command using the provided arguments returning the output of
//the command as a string.